shared/src/commonMain/kotlin/com/bunbeauty/shared/data/repository/SuggestionRepository.kt

```diff
@@ -0,0 +1,36 @@
+package com.bunbeauty.shared.data.repository
+
+import com.bunbeauty.shared.data.network.api.NetworkConnector
+import com.bunbeauty.shared.domain.model.Suggestion
+import com.bunbeauty.shared.domain.repo.SuggestionRepo
+import com.bunbeauty.shared.extension.dataOrNull
+
+class SuggestionRepository(
+    private val networkConnector: NetworkConnector,
+) : SuggestionRepo {
```

Comment:
why not CacheListRepository

Reply:
1. we don't store Suggestions locally 
2. we are using Map for cache, not List